platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -697,7 +697,7 @@ class ProjectSchemaAccessorsIntegrationTest : AbstractKotlinIntegrationTest() {
         assertThat(
             result.output,
             allOf(
-                containsString("slf4j-api-1.7.25.jar"),
+                containsString("slf4j-api-2.0.17.jar"),
```

Comment:
💅 This can either be reverted or synced with line 676
https://github.com/gradle/gradle/blob/4de012810ce73f115e00048cbdce5cb1aeef36f7/platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/ProjectSchemaAccessorsIntegrationTest.kt#L676